Demos/02-add-aad-auth/graph-tutorial/TokenStorage/SessionTokenStore.cs

```diff
@@ -118,7 +121,7 @@ private string GetUsersUniqueId(ClaimsPrincipal user)
                 var userObjectId = user.FindFirst("http://schemas.microsoft.com/identity/claims/objectidentifier").Value ??
                     user.FindFirst("oid").Value;
 
-                var userTenantId = user.FindFirst("http://schemas.microsoft.com/identity/claims/tenantid").Value ??
+                var userTenantId = user.FindFirst("http://schemas.microsoft.com/identity/claims/objectidentifier").Value ??
```

Comment:
This isn't correct, this value should be `tenantId`. See https://github.com/microsoftgraph/msgraph-training-aspnetmvcapp/pull/35